src/api.js

```diff
@@ -24,4 +24,28 @@ const fetchAllTopics = () => {
     })
 }
 
-export { fetchAllArticles, fetchAllTopics, fetchArticleByID };
\ No newline at end of file
+const sendUpVote = (article_id, votes) => {
+    return axios.patch(`https://cr-nc-news.herokuapp.com/api/articles/${article_id}`,
+    {
+        inc_votes: votes
+    }
+    ).then((res)=>{
+        return res;
+    }).catch((err) => {
+        console.log(err);
+    })
+};
+
+const sendDownVote = (article_id, votes) => {
+    return axios.patch(`https://cr-nc-news.herokuapp.com/api/articles/${article_id}`,
+    {
+        inc_votes: votes
+    }
+    ).then((res)=>{
+        return res;
+    }).catch((err) => {
+        console.log(err);
+    })
+};
+
```

Comment:
these look identical!

do you need them to be seperate?

think about a refactor here